2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Configure stats pid to 20568
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Loading settings from C:\Users\wangyu\.config\wandb\settings
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Loading settings from D:\paper\project\CENet-main\wandb\settings
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': 'd:\\paper\\project\\CENet-main\\train.py', 'program': 'd:\\paper\\project\\CENet-main\\train.py'}
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:_log_setup():524] Logging user logs to D:\paper\project\CENet-main\wandb\run-20231228_160040-5582fjsv\logs\debug.log
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:_log_setup():525] Logging internal logs to D:\paper\project\CENet-main\wandb\run-20231228_160040-5582fjsv\logs\debug-internal.log
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():564] calling init triggers
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():572] wandb.init called with sweep_config: {}
config: {}
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():614] starting backend
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():618] setting up manager
2023-12-28 16:00:40,757 INFO    MainThread:20568 [backend.py:_multiprocessing_setup():107] multiprocessing start_methods=spawn, using: spawn
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():624] backend started and connected
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():716] updated telemetry
2023-12-28 16:00:40,757 INFO    MainThread:20568 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2023-12-28 16:00:41,842 WARNING MsgRouterThr:20568 [router.py:message_loop():77] message_loop has been closed
2023-12-28 16:00:45,858 ERROR   MainThread:20568 [wandb_init.py:init():1188] transport failed
Traceback (most recent call last):
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\wandb_init.py", line 1170, in init
    run = wi.init()
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\wandb_init.py", line 755, in init
    cancel=True,
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\lib\mailbox.py", line 281, in wait
    raise MailboxError("transport failed")
wandb.sdk.lib.mailbox.MailboxError: transport failed
